SmartSearch.LuceneNet/Internals/SpecializedFields/HigherRelevanceLiteralField.cs

```diff
@@ -0,0 +1,66 @@
+﻿using SmartSearch.Abstractions;
+using SmartSearch.LuceneNet.Analysis;
+using System;
+using System.Globalization;
+using System.Text;
+
+namespace SmartSearch.LuceneNet.Internals.SpecializedFields
+{
+    internal class HigherRelevanceLiteralField : Field, ISpecializedField
+    {
+        private const string Suffix = "_lit";
+
+        public bool AnalyzeField => true;
+        public string OriginalName { get; }
+        public new float RelevanceBoost => FieldRelevanceBoost.GetBoostValue(Relevance) * 3f;
```

Comment:
This causes the relevance to be 27f, since fields with Relevance.Higher come with a boost of 9f.